server/challenges/Challenge.ts

```diff
@@ -0,0 +1,30 @@
+export interface Points {
```

Comment:
We should rename this file to types.ts which [is the standard](https://stackoverflow.com/questions/36633033/how-to-organize-typescript-interfaces).